monitoring/remote_monitoring_demo/recommendations_infra_demo/README.md

```diff
@@ -6,7 +6,7 @@ This script facilitates recommendation generation and validation through various
 - Data-Driven Recommendations: Utilize existing data from ROS or in-house benchmarks. (--dataDrivenRecommendations)
 - Cluster Monitoring: Gather live metrics for a specified cluster. (--monitorRecommendations)
 - Demo Benchmark: Use metrics from demo benchmarks. (--demoBenchmark)
-- Validation: Match recommendations with existing data. (--validate)
+- Validation: Match recommendations and boxplots with existing data. (--validate)
```

Comment:
@kusumachalasani - Can you document the scenarios tested here corresponding to the csvs used.

Reply:
I get this error on running
```Aggregating the data for file :  ./recommendations_demo/validateResults/advisor-backend-api_cpu-underprovision.csv
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/csubrama/github/chandra/kusuma/autotune-demo/monitoring/remote_monitoring_demo/recommendations_infra_demo/recommendations_demo/recommendation_validation.py", line 178, in aggregateWorkloads
    agg_df = agg_df.append(df)
             ^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/site-packages/pandas/core/generic.py", line 6296, in __getattr__
    return object.__getattribute__(self, name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DataFrame' object has no attribute 'append'. Did you mean: '_append'?
sed: can't read metrics.csv: No such file or directory
sed: can't read metrics.csv: No such file or directory

```
Is there something that I 'm missing here?

Reply:
This is because of the version incompatibilities. Monitoring demo setup should be done with the versions mentioned in requirements.txt which is called as part of the script.
https://github.com/kusumachalasani/autotune-demo/blob/validateboxplots/monitoring/remote_monitoring_demo/recommendations_infra_demo/recommendations_demo.sh#L163

Reply:
'append' function exists only in pandas==1.1.5

Reply:
Run through jenkins failed with this error:

```
File "/home/jenkins/kruize-workspace/autotune-demo/monitoring/remote_monitoring_demo/recommendations_infra_demo/recommendations_demo/recommendation_validation.py", line 93, in aggregateWorkloads
    df = df.dropna(subset=columns_to_check, how='any')
  File "/home/jenkins/.local/lib/python3.9/site-packages/pandas/core/frame.py", line 5004, in dropna
    raise KeyError(list(np.compress(check, subset)))
KeyError: ['owner_kind', 'owner_name', 'workload', 'workload_type']
sed: can't read metrics.csv: No such file or directory
sed: can't read metrics.csv: No such file or directory
```

Reply:
Fixed the issue. 
Run has passed now - https://ci.app-svc-perf.corp.redhat.com/job/ExternalTeams/job/Autotune/job/kruize_reco_validation_tests/23/console

Reply:
Thanks Kusuma. Can you suppress any unwanted logging, the console output of the job on jenkins is not displayed due to a lot of data.

Reply:
Commented out response texts when calling the kruize APIs. Keep them in the file as we can uncomment them for any debugging purposes.